pkg/config/config.go

```diff
@@ -57,6 +57,11 @@ type Config struct {
 	// DebugFileDirectories is the list of directories Delve will use
 	// in order to resolve external debug info files.
 	DebugInfoDirectories []string `yaml:"debug-info-directories"`
+
+	// FullPathMatch will try to eval all file paths to realpath if location contains filename.
```

Comment:
"When FullPathMatch is true Delve will try to resolve symbolic links when setting breakpoints. Note: this option may cause performance degradation."

Reply:
@aarzilli en, maybe just `setting breakpoints` is accurate, ex : `list` cmd. 
Need other description, how about changing `setting breakpoints` into `searching source files` above?

Reply:
Have changed the description of `FullPathMatch`, please check again, thx very much.

Reply:
This is fine.